test/Atmos/Parameterizations/Microphysics/Tot_N_Act_test.jl

```diff
@@ -0,0 +1,30 @@
+Pkg.add("SpecialFunctions")
+@testset "total_N_Act_test" begin
```

Comment:
It's amazing how many tests you implemented. This is really great. I think we should move them to their own folder. Let's say named `test/Atmos/Parameterizatons/Activation/`